src/test/java/com/google/api/codegen/testdata/library_gapic.yaml

```diff
@@ -167,6 +168,7 @@ interfaces:
     page_streaming:
       request:
         token_field: page_token
+        page_size_field: page_size
```

Comment:
Ah -- I had intended ListStrings to be a test example for Python where `page_size` is in the proto but not the config. In this case, Python codegen should just treat `page_size` like an ordinary optional parameter. Since there are three page streaming requests, how about I remove `page_size` from a different one?